### DISCUSSION TITLE
how to update from 0.6.1 to 0.6.3 without setting everything back up?

### DISCUSSION THREAD

**Discussion by ghost** [0xPolygon/polygon-edge#1136] [General]
# [ Subject of the issue ]
## Description I'd like to install the latest updates to my running chain, but don't want to use the following line, as I don't know if it will overwrite the existing validator settings (Private key, Node ID, BLS key)

git clone https://github.com/0xPolygon/polygon-edge.git

Describe your issue in as much detail as possible here. As above.

## Your environment
- OS and version. Ubuntu 20.04
- The version of the Polygon Edge.    0.6.1
  (*Confirm the version of your Polygon edge client by running the following command: `polygon-edge version --grpc-address GRPC_ADDRESS`*)
- The branch that causes this issue.
- Locally or Cloud hosted (which provider). Cloud (AWS, Ionos & private servers) 
- Please confirm if the validators are running under containerized environment (K8s, Docker, etc.). No container

## Steps to reproduce
- Tell us how to reproduce this issue.
- Where the issue is, if you know.
- Which commands triggered the issue, if any.
- Provide us with the content of your genesis file.
- Provide us with commands that you used to start your validators.
- Provide us with the peer list of each of your validators by running the following command: `polygon-edge peers list --grpc-address GRPC_ADDRESS`.
- Is the chain producing blocks and serving customers atm?

## Expected behavior
- Tell us what should happen. A nice easy upgrade to 0.6.3
- Tell us what happened instead. Nothing yet, as I don't want to run something and break it.

**Comment by laviniat1996** [comment]
Hi @Nexsuperne101 ,

You can update by just stopping the nodes and changing the binaries, without any alterations to your chain setup. Full guide here on the update procedure here:  https://wiki.polygon.technology/docs/edge/validator-hosting#update
